src/components/MessageLogger/MessageList.module.scss

```diff
@@ -0,0 +1,17 @@
+@use "@styles/overrideDefaultMethods.scss";
+@use "@styles/styles.scss";
+
+#containerMessages {
+  background-color: styles.$default-background;
+  width: 35rem;
```

Comment:
set width and height to 100%, as the actual size comes from the SpliLayout component